src/js/defaults.js

```diff
@@ -0,0 +1,18 @@
+import {validation as v} from './utils';
```

Comment:
I would not use `import as` otherwise you would always need to go up to the imports to see what these shorthand versions are.

Reply:
ok